config.cmake.template

```diff
@@ -50,13 +50,34 @@
 # set(BDDCML_ROOT "$ENV{HOME}/local/BDDCML/bddcml-2.2")
 
 ### Python setting ###############################################################
-# USE_PYTHON - only use python if this is set to "yes"
+# USE_PYTHON - use embedded python if this is set to "yes"
 # We add python library only on explicit request since it leads to tons of errors when debugging with valgrind.
 #
 # set(USE_PYTHON "yes")
 
 # PYTHON_COPY - make copy of python files, necessary for release builds
 # set(PYTHON_COPY "yes")
+#  - Following variables MUST ALSO BE SET:
+#    - PYTHON_ROOT
+#           - absolute path to python root e.g.:
+#             set(PYTHON_ROOT "/usr/lib/python2.7")
+#           
+#    - PYTHON_COPY_PACKAGES 
```

Comment:
Comments are unclear. In particular the difference between PYTHON_COPY_PACKAGES and PYTHON_SYSPATH 
is not clear.